package.json

```diff
@@ -43,6 +43,8 @@
     "build:bundle": "webpack --progress --bail --mode production",
     "build:bundle-stats": "webpack --progress --bail --mode production --json > webpack-stats.json",
     "dist": "scripts/package.sh",
+    "docker": "docker build -t pegacorn-communicate-app-web .",
+    "lingo-web-docker": "docker run -p 3000:80 pegacorn-communicate-app-web",
```

Comment:
This issue no longer exists as edit was done on readme in later commits to this commit.

Reply:
The command will run smoothly given that we dont need to pass parameters or build arguements but this seems to be removed already throughout the time since I made this commit.